docs/antora.yml

```diff
@@ -0,0 +1,5 @@
+name: opa
+version: master
```

Comment:
If it refers to the branch, it should be "main". I checked some other operators and it is "master" everywhere, so if im wrong just ignore this.

Reply:
Yeah, `master` is not the branch but an Antora specific thing.

Reply:
Ok did not know that ;)